routing.go

```diff
@@ -162,17 +162,21 @@ func NewRouter(c config.ConfigReader) (rest.App, error) {
 		new(deploymentsView.DeploymentsView))
 	limitsController := limitsController.NewLimitsController(limitsModel,
 		new(view.RESTView))
+
+	tenantsDeploymentController := tenantsController.NewDeploymentController(deploymentModel)
```

Comment:
there is already instance of this object available `tenantsController`,  no need for another one